go.mod

```diff
@@ -39,27 +39,56 @@ require (
 	go.opencensus.io v0.22.1 // indirect
 	golang.org/x/crypto v0.0.0-20191107222254-f4817d981bb6
 	golang.org/x/net v0.0.0-20191108063844-7e6e90b9ea88 // indirect
-	golang.org/x/sys v0.0.0-20191105231009-c1f44814a5cd // indirect
 	golang.org/x/time v0.0.0-20191024005414-555d28b269f0 // indirect
 	golang.org/x/tools v0.0.0-20191107235519-f7ea15e60b12 // indirect
 	google.golang.org/appengine v1.6.5 // indirect
 	google.golang.org/grpc v1.25.0 // indirect
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
 	gopkg.in/ini.v1 v1.51.0 // indirect
 
-	gopkg.in/yaml.v2 v2.2.7
-	k8s.io/api v0.0.0-20190831074750-7364b6bdad65
-	k8s.io/apimachinery v0.0.0-20190831074630-461753078381
-	k8s.io/client-go v0.0.0-20190819141724-e14f31a72a77
-	k8s.io/cluster-bootstrap v0.0.0-20190831080953-99cb41cb5d35
+	gopkg.in/yaml.v2 v2.2.8
```

Comment:
See https://github.com/flanksource/platform-operator/blob/master/go.mod - importing the kubernetes api's can be done with a semantic version now